<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TalkView</class>
 <widget class="QWidget" name="TalkView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#TalkView{
background-color: rgb(233, 238, 244);
}</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>500</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit#textEdit{
border:0px;
background-color:transparent;
}

QTextEdit#textEdit:focus{
border:0px;
	background-color: rgb(255, 255, 255);
}
</string>
   </property>
  </widget>
  <widget class="Mylabel" name="phizBar">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>260</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/images/widgets/phiz.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Mylabel" name="imgBar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/images/widgets/picture.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Mylabel" name="timeBar">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>260</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/images/widgets/time.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>257</y>
     <width>60</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#sendButton{
background-color: rgb(24, 180, 237);
	color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QQuickWidget" name="bubbleList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>501</width>
     <height>210</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
  </widget>
  <widget class="Mylabel" name="backBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>7</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/images/widgets/back.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Mylabel" name="friendName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Mylabel" name="menuBar">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>7</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/images/widgets/menu.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>Mylabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
